app/page.tsx

```diff
@@ -1,3 +1,54 @@
-export default function Home() {
-  return <main className=''>Home</main>;
+import Image from 'next/image';
+import Link from 'next/link';
+
+import { MotionDiv } from '@/components/Motion';
+import {
+  fetchProjects,
+  fetchSpecificSkills,
+} from '@/sanity/lib/utils';
+import { mainSkills } from '@/constants';
+import GetImage from '@/sanity/lib/getImage';
+import { ImageComponent, urlFor } from '@/sanity/lib/sanity';
+
+export default async function Home() {
+  const skills = await fetchSpecificSkills(mainSkills);
+  console.log(skills);
+  return (
+    <main className=''>
```

Comment:
```suggestion
    <main>
```
Clean up empty classnames. Remember a PR is a request to have your code merged into main. It shouldn't have errors, commented out code, logging from testing, unused imports, or empty/unused props.